src/components/SelectInput/SelectInput.module.scss

```diff
@@ -0,0 +1,72 @@
+@import '../../styles/variables.scss';
+
+:global {
+  .selectInputWrapper {
+    .selectInput {
+      .selectInput__control {
```

Comment:
There's no use in having this a CSS module if the entire file is wrapped in :global. I would try one of these options:

```
.otherClassName {
  composes: className from "./style.css";
}
```

```
.otherClassName {
  composes: globalClassName from global;
}
```

Reply:
Yup, agreed, this one is still in progress, and not working properly. Adding the global was the last step I did last night to get it working properly but it still isn't properly passing the styles down to our react-select. I plan to not have it be a module altogether and just import the styles.